core/modules/user/user.module

```diff
@@ -1720,6 +1739,11 @@ function user_admin_paths() {
     'user/*/edit' => TRUE,
     'user/*/edit/*' => TRUE,
   );
+  if (config_get('system.core', 'user_login_appearance_setting') == 'simplified') {
+    $paths[] = 'user/login';
+    $paths[] = 'user/password';
+    $paths[] = 'user/register';
+  }
```

Comment:
This approach won't work for anonymous users on most sites because they do not have the "view administration theme" permission (which is a dumb permission to have but :man_shrugging:).

Unless you made that decision intentionally, in which case new installations should grant that permission out of the box. On the current sandbox sites, the simplified login pages are being rendered by Basis because of this.

![image](https://github.com/user-attachments/assets/c43b9c72-534c-4eed-b386-ad4e20bd8e3e)

I would be tempted to go back to `hook_custom_theme()` to side-step that rather thorny problem. See this section of @docwilmot's PR: https://github.com/backdrop/backdrop/pull/4750/files#r1894777974

Reply:
I can confirm, that the admin theme isn't used now. Not sure, why it should be, anyway, as the simplified pages also look good in the frontend theme (be it Basis or any other). But I don't want to derail the discussion. :wink:

This works for me:
```php
function user_custom_theme() {
  $config = config('system.core');
  if ($config->get('user_login_appearance_setting') == 'simplified') {
    return $config->get('admin_theme');
  }
}

```

Reply:
An example that works for me:

```php
function user_custom_theme() {
  if (!in_array(current_path(), array('user/login', 'user/password', 'user/register'))) {
    return;
  }
  $config = config('system.core');
  if ($config->get('user_login_appearance_setting') == 'simplified') {
    return $config->get('admin_theme');
  }
}


```

Reply:
Yeah, it gets tricky. I suggested making a second option on the Appearance page in https://github.com/backdrop/backdrop-issues/issues/6570#issuecomment-2564640596, but then that conflicts with the "Tabs" / "No Tabs" options (which aren't guaranteed to be safe in the admin theme/layout). We could solve this problem later. For now, making the simplified login form only use the admin theme (as it was in docwilmot's PR) seems like the easier and existing compromise.

Reply:
OK, I brought back `user_custom_theme()`. Sorry to miss that in the discussion.